src/components/RepositoryReleasesChangelog.tsx

```diff
@@ -49,7 +49,15 @@ const ReleaseChangelogGroup = ({
   return (
     <Box key={title}>
       {shouldShowTitle && (
-        <Heading as="h2" size="xl" mb={4} textTransform={textTransform}>
+        <Heading
+          as="h2"
+          size="xl"
+          bg="white"
+          mb={4}
+          py={4}
+          textTransform={textTransform}
+          sx={{ position: 'sticky', top: '0px' }}
```

Comment:
[The `sx` property is considered an escape hatch](https://chakra-ui.com/docs/features/the-sx-prop#use-cases) since all basic CSS properties are available as component properties 😅. This comes from Styled System, you can read more about it [here](https://styled-system.com/).

You must set these styles through [Position props](https://chakra-ui.com/docs/features/style-props#position):

```suggestion
          position="sticky" // `pos` is also valid
          top={0}
```

Reply:
Ah okay. I just searched through the Chakra UI repository to see if anybody had any tips for sticky positioning, and somebody showed that `sx` thing. 😄 

I agree, we should stay within the Chakra UI system as much as possible. 😄